docs/releases/5.0.txt

```diff
@@ -401,7 +401,8 @@ Management Commands
 Migrations
 ~~~~~~~~~~
 
-* ...
+* Migrations now support serialization of functions decorated with
```

Comment:
On a second thought, I think it would be a good idea to remove "Migrations now" since the section title is specific about Migrations. So:

```suggestion
* Support for serialization of functions decorated with... was added.
```

Reply:
I copied from the 4.2 release notes where `EnumFlag` was added. Based on that would you still prefer the change?

Reply:
I made the suggested change anyway.